src/app/branded/step-1/branded-checkout-step-1.tpl.html

```diff
@@ -46,10 +90,26 @@ <h3 class="panel-name" translate>{{'PAYMENT'}}</h3>
   </div>
   <div class="panel">
     <div class="panel-body text-right">
-      <button class="btn btn-primary"
+      <div class="checkout-cta pull-right" ng-if="$ctrl.useV3 === 'true'">
+       <recaptcha-wrapper
+       
+              action="'branded_submit'"
+              on-success="$ctrl.submit"
+              on-failure="$ctrl.handleRecaptchaFailure"
+              component-instance="$ctrl.selfReference"
```

Comment:
You should be able to do this and not need to set `this.selfReference` in the controller.

```suggestion
              component-instance="$ctrl"
```